website/blog/2019-06-26-7.5.0.md

```diff
@@ -10,33 +10,34 @@ share_text: "Babel 7.5.0 Released"
 
 Today we are releasing Babel 7.5.0!
 
-This release includes improved support for a few ECMAScript proposals: you can now try the "F#" variant of the Stage 1 pipeline operator, and we now provide an official plugin and `preset-env` support for the Stage 4 dynamic `import()` proposal.
-It also has experimental support for TypeScript `namespace`s, and for Browserslist's `default` query in `preset-env`.
+This release includes improved support for a few ECMAScript proposals: in particular "F#" variant of the Stage 1 pipeline operator  and now an official plugin for the Stage 4 dynamic `import()` proposal (along with `preset-env` support).
+It also has experimental support for TypeScript `namespace`s and for Browserslist's `default` query in `preset-env`.
 
 You can read the whole changelog [on GitHub](https://github.com/babel/babel/releases/tag/v7.5.0).
 
 <!-- truncate -->
 
 Thanks to [Wesley Wolfe](https://github.com/Wolvereness), [Martin Zlámal](https://github.com/mrtnzlml), [Blaine Bublitz](https://github.com/phated), [Letladi Sebesho](https://github.com/Letladi), and [Paul Comanici](https://github.com/darkyndy) for their first PRs!
-We also want to welcome the newest addition to the Babel organization: [Tan Li Hau](https://github.com/tanhauhau)! Li Hau has been costantly helping us with bug fixes by triaging issues; you can read his [blog](https://lihautan.com/blogs) and follow him on Twitter ([@lihautan](https://twitter.com/lihautan)).
 
 We are always looking for help, especially with triaging issues, reviewing pull requests and helping people on Slack. We are experimenting with the new [triage](https://github.blog/changelog/2019-05-23-triage-and-maintain-roles-beta/) role implemented by GitHub, and we want to recognize people people from the community who will stand up helping us! 😉
 
-Our work has made been possible also by our sponsors. We want to thank [Discord](https://discordapp.com/) and [clay](https://clay.global/) for becoming Silver Sponsors on [OpenCollective](https://opencollective.com/babel), and [Linkedin](https://linkedin.com/) for becoming a Silver Sponsor of Henry on GitHub!
+Thus we also want to welcome our newest addition to the Babel organization: [Tan Li Hau](https://github.com/tanhauhau)! Li Hau has been a great help in constantly triaging issues and fixing bugs; please check out his [blog](https://lihautan.com/blogs) and follow him on Twitter ([@lihautan](https://twitter.com/lihautan)).
```

Comment:
Doesn't this sounds too much like if we were imploring our users to follow @tanhauhau? :stuck_out_tongue:

Reply:
🙈